src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Checkbox.js

```diff
@@ -25,9 +26,21 @@ export default class Checkbox extends React.Component<FieldTypeProps<boolean>> {
         } = this.props;
 
         if (type === 'toggler') {
-            return <Toggler checked={!!value} onChange={this.handleChange}>{label}</Toggler>;
+            return (
+                <Toggler
+                    checked={!!value}
+                    disabled={!!disabled}
+                    onChange={this.handleChange}
+                >{label}</Toggler>
```

Comment:
That looks weird to me... if we have such a situation `{label}` usually goes into an own line...